pandas/tests/test_strings.py

```diff
@@ -430,6 +430,14 @@ def test_replace(self):
         result = values.str.replace("(?<=\w),(?=\w)", ", ", flags=re.UNICODE)
         tm.assert_series_equal(result, exp)
 
+        # GH 13438
+        for pdClass in (Series, Index):
+            for repl in (None, 3, {'a': 'b'}):
+                for data in (['a', 'b', None], ['a', 'b', 'c', 'ad']):
+                    with self.assertRaises(TypeError):
+                        values = pdClass(data)
```

Comment:
Sorry don't quite get you. Should I entirely remove the with block and replace it with something else?